### DISCUSSION TITLE
[Problem/Bug]: Update wv2 release docs, to manage expectations about both windows/MacOs platform

### DISCUSSION THREAD

**Discussion by omonimus1** [MicrosoftEdge/WebView2Feedback#4355] [Tips and Tricks]
[Problem/Bug]:  Can the release doc being updated, to know if a new wv2 API will be supported also on mac, when estimated (more or less), or if already enabled too
----------------
is too time consuming to check headers in both windows/macOs platforms, to verify if a new api is supported also on MacOs.
Release notes should now manage expectations for BOTH platforms, in terms of support and/or incoming/expected availability.

**Comment by victorhuangwq** [comment]
We are tracking that request here: https://github.com/MicrosoftEdge/WebView2Feedback/issues/1314. 
We will update there with any information about it.
